runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/blog/BreadcrumbsSection/BreadcrumbsSection.module.scss

```diff
@@ -1,5 +1,49 @@
+@import '#src-landing/scss/main.scss';
+
+$SECTION-HEIGHT: 59px;
+$SEPARATOR_SPACE: $padding-1;
+
 .root {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
     grid-column: span 2;
     width: 100%;
-    height: 59px;
+    height: $SECTION-HEIGHT;
+    margin-top: $padding-2;
+    margin-bottom: $padding-2;
+}
+
+.breadcrumb {
+    display: flex;
+}
+
+.link {
+    position: relative;
+    text-decoration: none;
+
+    &:hover::after {
+        content: '';
+        position: absolute;
+        bottom: -3px;
+        display: block;
+        width: 100%;
+        height: 2px;
+        background-color: $palette-accent-primary;
+    }
+}
```

Comment:
oh so thats how you do it. Imo its too complex for what it does. See `BlogSharePanel` I just made a bottom border to be present all the time and only on hover it gets its color otherwise its transparent